### DISCUSSION TITLE
"jax.numpy.bool" seems to be removed now

### DISCUSSION THREAD

**Discussion by Mio826** [netket/netket#1961] [Q&A]
when I ran this file: netket/netket/sampler/rules/exchange.py
an error occured : 
![图片](https://github.com/user-attachments/assets/80a005d8-2129-45b8-a69b-8aa08e8db20e)
It seems like new version of Jax removed "jnp.bool"

**Comment by PhilipVinc** [comment]
what version of jax do you have?
I think it does still have it

**Reply by Mio826** [reply]
my version is 0.4.20.
I changed "jnp.bool" to "jnp.bool_", it seems to work now :)
